web/views/note_view.ex

```diff
@@ -1,4 +1,12 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  defp sanitize_html(unsafe) do
+    {:safe, HtmlSanitizeEx.html5(unsafe)}
```

Comment:
The `html5` says it is "not yet ready in any way". https://github.com/rrrene/html_sanitize_ex/blob/master/lib/html_sanitize_ex/scrubber/html5.ex

Just a consideration. Any reason you chose it over `basic_html`?